app/jobs/story_update_job.rb

```diff
@@ -20,7 +20,7 @@ def receive_story_update(data)
     load_resources(data)
     episode ? update_episode : create_episode
```

Comment:
So this is getting create/update messages for ALL stories, not just ones we know are in a podcast-distributed series, right?  Couldn't this result in `episode` and `podcast` being undefined on the next line, if the story has no series?

Also seems like the error thrown from `Podcast.find_by!(prx_uri: series_uri)` would just bubble up through here.

... correct me if i'm way off here.